ci-operator/config/openshift/aws-ebs-csi-driver/openshift-aws-ebs-csi-driver-release-4.18.yaml

```diff
@@ -78,6 +78,24 @@ tests:
   steps:
     cluster_profile: aws-qe
     workflow: openshift-e2e-aws-csi-extended
+- as: e2e-aws-csi-techpreview-volumeattributesclass
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      TEST_ARGS: --run=VolumeAttributesClass
+      TEST_SUITE: all
+    workflow: openshift-e2e-aws
+- as: periodic-e2e-aws-csi-techpreview-volumeattributesclass
```

Comment:
I think the periodic job should be only in `-master.yaml`. See https://github.com/openshift/release/tree/master/ci-operator/config/openshift/csi-operator, automation removed periodic jobs from release-4.18 and 4.19 files.